nucypher/policy/conditions/time.py

```diff
@@ -15,9 +15,12 @@ class TimeCondition(ReencryptionCondition):
     METHOD = 'timelock'
 
     class Schema(CamelCaseSchema):
-        name = fields.Str()
-        method = fields.Str(dump_default="timelock")
-        return_value_test = fields.Nested(ReturnValueTest.ReturnValueTestSchema())
+        SKIP_VALUES = (None,)
+        name = fields.Str(required=False)
+        method = fields.Str(dump_default="timelock", required=True)
```

Comment:
Ultimately I think this will be better expressed in a dedicated field ie `conditionType`.  Then we can clean up the jenky contract type resolution in `nucypher.policy.utils,resolve_condition_lingo`.

Reply:
I agree with the strategy. Let's do that in a follow-up.

Reply:
Agreed, it's outside the scope of this PR.